pkgs/applications/networking/filebrowser/default.nix

```diff
@@ -0,0 +1,55 @@
+{ buildGoModule, buildNpmPackage, fetchFromGitHub, lib }:
+
+let
+  frontend = buildNpmPackage rec {
```

Comment:
This derivation should be part of the internal one as a `passthru` for example so you can do `filebrowser.frontend` or `filebrowser.ui`, look nixpkgs for existing examples.

Reply:
Added a passthru, please take a look